hammer_cli_reference/README.adoc

```diff
@@ -0,0 +1,31 @@
+= Hammer CLI reference
+
+foreman-documentation contains a Hammer CLI guide for all build flavours.
+The guide contains a list of reference files that are based on Hammer CLI help output for a specific Foreman+Katello version and, optionally, Foreman+Katello plugins.
+
+In *downstream* docs, each team maintains its own set of downstream-only commits that overwrite the assembly and add necessary reference files.
+In *upstream* docs, the community can update the reference files based on a Foreman+Katello instance.
+
+== Updating Hammer CLI reference files
+
+.Prerequisites
+* You have access to a Foreman+Katello instance.
+* You run Ansible on your local machine.
+
+.Limitations
+* Tested with Foreman 3.11/Katello 4.13 on EL8 only
+
+.Procedure
+. Ensure that there are no uncommitted changes in `foreman-documentation`.
+. Set the FQDN of your Foreman+Katello instance in `hammer/inventory.yaml`.
```

Comment:
```suggestion
. Set the FQDN or IP address of your Foreman+Katello instance in `hammer_cli_reference/inventory.yaml`.
```
There is no `hammer/` folder.
IP works as well.

Reply:
Applied.